packages/cli/tsconfig.json

```diff
@@ -10,7 +10,8 @@
     "sourceMap": true,
     "esModuleInterop": true,
     "strict": true,
-    "lib": ["es2015", "es6", "esnext.asynciterable", "dom"]
+    "lib": ["es2015", "es6", "esnext.asynciterable", "dom"],
+    "resolveJsonModule": true
```

Comment:
This is needed in order to read JSON files

Reply:
makes sense